docs/install.md

```diff
@@ -109,6 +109,20 @@ You can also temporarily switch to a specific version of Nextflow with the `NXF_
 NXF_VER=23.10.0 nextflow info
 ```
 
+## Seqera Platform
+
+Take your Nextflow usage to the next level with [Seqera Platform](https://seqera.io/platform/), a tool created by the team behind Nextflow that's designed to monitor, manage,
```

Comment:
"Take your Nextflow usage to the next level" doesn't really match the tone of the rest of the docs, feels more like a marketing pitch than something purely informative.

Since this is the install page, maybe the focus should be on the fact that you don't need to install Nextflow locally this way. I mean leading with that rather than saying it further down

Reply:
Agreed, removed this sentence.